lib/data-types.js

```diff
@@ -465,7 +465,7 @@ DATE.prototype._stringify = function _stringify(date, options) {
 function DATEONLY() {
   if (!(this instanceof DATEONLY)) return new DATEONLY();
 }
-util.inherits(DATEONLY, ABSTRACT);
+inherits(DATEONLY, ABSTRACT);
```

Comment:
No, it's not needed at all, I've simply noticed it hadn't been changed like other inherits calls. I've removed its commit from this PR.